core/camel-core/src/test/java/org/apache/camel/processor/ThrottlerTest.java

```diff
@@ -24,24 +24,19 @@
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.component.mock.MockEndpoint;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
 
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
 public class ThrottlerTest extends ContextTestSupport {
     private static final int INTERVAL = 500;
     private static final int TOLERANCE = 50;
     private static final int MESSAGE_COUNT = 9;
 
-    protected boolean canTest() {
-        // skip test on windows as it does not run well there
-        return !isPlatform("windows");
-    }
-
+    @DisabledOnOs(OS.WINDOWS)
```

Comment:
As all tests seem to be disabled for windows, maybe we could add the annotation only once at the class definition level?

Reply:
Change for this and some other cases